dvc/scm/git/__init__.py

```diff
@@ -33,13 +33,16 @@ class Git(Base):
     GIT_DIR = ".git"
 
     def __init__(self, root_dir=os.curdir, repo=None):
+        """Git class constructor.
+        Requires `Repo` class from `git` module (from gitpython package).
+        """
         super(Git, self).__init__(root_dir, repo=repo)
 
-        import git
+        from git import Repo
         from git.exc import InvalidGitRepositoryError
 
         try:
-            self.git = git.Repo(root_dir)
+            self.repo = Repo(root_dir)
```

Comment:
Is this git -> repo renaming necessary? Also, you are overwriting `self.repo` that is a dvc repo and set in SCMBase.

Reply:
Thanks about the heads up about dvc.scm.base.repo! I don't think its used though, is it?

I still suggest renaming it though, to give it a better name that describes what the property is, and because otherwise we end up using `self.git.git.` or `dvc.smc.git.git` in other places, which is more difficult to decipher from the PoV of a developer new to the code base.

However other objects also have `self.git = Repo(...` so I guess I would have to rename all of the occurrences in all the files to be consistent... Up to you @efiop @shcheklein , please lmk, thanks.

Reply:
@jorgeorpinel Let's not do that in this PR, since it is out of the scope.

Reply:
Change undone in 98f461d